### DISCUSSION TITLE
[@types/globaloffensive] getCasketContents doesn't return promise

### DISCUSSION THREAD

**Discussion by DarkGL** [DefinitelyTyped/DefinitelyTyped#64582] [Issues with a @types package] [answered]
getCasketContents doesn't return promise

but in typings we have code like this:

`getCasketContents(
        casketId: string,
        callback: (err: Error | null, items: GlobalOffensive.InventoryItem[]) => void,
    ): Promise<GlobalOffensive.InventoryItem[]>;`

[a comment by an automated account is not shown]

**Comment by joshuajeschek** [comment, marked as answer]
Good catch!
fixed it and also updated the package to the newest version in #64607

**Reply by joshuajeschek** [reply]
merged #64607
